en/SQLDatabase.md

```diff
@@ -53,4 +54,4 @@ Otherwise you might override someone else's work.
 
 ## Try it out
 
-You can test the shared database support by using <https://www.freemysqlhosting.net>.
\ No newline at end of file
+You can test the shared database support by using <https://www.freemysqlhosting.net>.
```

Comment:
I just edited this directly. I just merged the other PR in, did not see this one here.